sdk/healthinsights/test-resources-post.ps1

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Allow time for the resource to be created
+
+Start-Sleep -s 300
```

Comment:
Is there a way we can poll the resource creation state via azure powershell rather than waiting for 5 minutes?